resources/datadog-integrations/software.rb.erb

```diff
@@ -8,6 +8,7 @@ if windows?
   require_relative "../../resources/datadog-integrations/validate_manifest"
 else
   require_relative "<%= project_dir %>/resources/datadog-integrations/validate_manifest"
+  whitelist_file /\/opt\/datadog-agent\/.+\.so.+/
```

Comment:
Hmm I think the integration-specific deps are in `/opt/datadog-agent/integrations/<name>/lib` so it'd be good to still cover them I think.

AFAIK we only need to whitelist the ones in ``/opt/datadog-agent/embedded` since we assume that the agent build's healthcheck has already taken care of those.